Tests/Functional/Controller/RedirectControllerTest.php

```diff
@@ -18,23 +18,39 @@
 
 class RedirectControllerTest extends BaseTestCase
 {
-    const ROUTE_ROOT = '/test/routing';
+    const ROUTE_ROOT = '/test/routing-functional';
 
     /**
-     * @var \Symfony\Cmf\Bundle\RoutingBundle\Controller\RedirectController
+     * @var RedirectController
      */
     protected $controller;
 
-    public function setUp()
+    /**
+     * Creates the route root.
+     */
+    protected function setUp()
     {
-        parent::setUp();
-        $this->db('PHPCR')->createTestNode();
+        $this->makeSureTestRootExists();
```

Comment:
`initTestNode` ? In anycase I would stick with the existing TestNode terminology instead of TestRoot.